spec/plans/results.fmf

```diff
@@ -119,6 +138,33 @@ example:
       guest:
         name: default-0
 
+  # Duplication warning: after updating the following examples, copy them into
+  # /spec/tests/result.fmf.
+  #
+  # The goal is to treat results.fmf as the main source of truth, and while
+  # it's useful to show custom results examples in test specification as well,
+  # let's strive to keep it in sync.
```

Comment:
Is this also a comment? Just pointing out that it's styled differently than the previous one which used `..` to write a comment.

Reply:
Yes, should work fine, this is a `yaml` comment.